components/google_drive/utils.mjs

```diff
@@ -81,6 +81,23 @@ async function getFileStream({
     : fs.createReadStream(filePath);
```

Comment:
Move this file to `common/utils.mjs`

Reply:
Maybe we should do this in a different PR? If I move these files probably we will have to test all actions and sources again.

Reply:
Of course!